### PR TITLE
Feature/refactor tables

### PR DESCRIPTION
**Pull request by aliashkevich**
Created universal components 'Table', 'ActionsTable' with Edit/Delete buttons, 'AddButton'. 
Removed unused package 'react-responsive'.
Added 'prop-types' package.
Refactored ClientsTable, ProjectsTable to use unified Table.

### CODE REVIEW

File: src/components/global/ActionsTable.js [hunk 1]
```diff
@@ -0,0 +1,75 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import './ActionsTable.css';
+
+function ActionsTable({...props}) {
```
Comment (crisanlucid) [inline src/components/global/ActionsTable.js]: what is the reason to have `...props`  if next line you deconstruct?

File: src/components/global/ActionsTable.js [hunk 2]
```diff
@@ -0,0 +1,75 @@
...
+ActionsTable.defaultProps = {
+  tableColor: 'primary',
+};
+
+ActionsTable.propTypes = {
+  tableName: PropTypes.string,
+  tableDescription: PropTypes.string,
+  tableHead: PropTypes.arrayOf(PropTypes.string),
+  tableData: PropTypes.arrayOf(PropTypes.arrayOf(PropTypes.any)),
+  tableColor: PropTypes.oneOf([
```
Comment (crisanlucid) [inline src/components/global/ActionsTable.js:65]: (thumb)

File: src/components/global/AddButton.js [hunk 3]
```diff
@@ -0,0 +1,16 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+export default function AddButton(props) {
+  const {buttonName} = props;
+  return (
+    <button type='button' className='btn btn-success'>
+      <i className='material-icons'>add_circle</i>
+      {' ' + buttonName}
```
Comment (crisanlucid) [inline src/components/global/AddButton.js]: use back tick   `

File: src/components/login/LoginForm.js [hunk 4]
```diff
@@ -0,0 +1,69 @@
...
+                name='password'
+                value={password}
+                onChange={handleInputChange}
+                className='form-control'
+                id='exampleInputPassword1'
+                placeholder='Password'
+                required
+              />
+            </div>
+            {/* <div className='form-check'>
```
Comment (crisanlucid) [inline src/components/login/LoginForm.js:52]: remove comments

File: src/pages/Login.js [hunk 5]
```diff
@@ -1,5 +1,5 @@
 import React, {Component} from 'react';
-import LoginForm from '../components/Login/LoginForm';
+import LoginForm from '../components/login/LoginForm';
```
Comment (crisanlucid) [inline src/pages/Login.js:2]: keep the dir with  Upercase  "Login"

File: src/pages/ProjectDetail.js [hunk 6]
```diff
@@ -1,6 +1,6 @@
 import React from 'react';
 import Header from '../components/Header';
-import ProjectDetailCard from '../components/Project/ProjectDetailCard';
+import ProjectDetailCard from '../components/projects/ProjectDetailCard';
```
Comment (crisanlucid) [inline src/pages/ProjectDetail.js]: same here

File: src/pages/Projects.js [hunk 7]
```diff
@@ -1,18 +1,45 @@
 import React from 'react';
 import Header from '../components/Header';
-import ProjectTable from '../components/Project/ProjectTable';
-import ProjectAdd from '../components/Project/ProjectAdd';
+import ProjectsTable from '../components/projects/ProjectsTable';
+import AddProjectButton from '../components/projects/AddProjectButton';
```
Comment (crisanlucid) [inline src/pages/Projects.js]: Projects

File: src/pages/Projects.js [hunk 8]
```diff
@@ -1,18 +1,45 @@
...
+
+  componentDidMount() {
+    fetch(this.props.url + '/projects')
+      .then(res => res.json())
+      .then(data => {
+        this.setState({
+          projects: data.projects,
+        });
+      })
+      .catch(error => console.log(error));
```
Comment (crisanlucid) [inline src/pages/Projects.js:27]: basic validation:
`alert(error)`

File: src/pages/Projects.js [hunk 9]
```diff
@@ -1,18 +1,45 @@
...
+  constructor(props) {
+    super(props);
+
+    this.state = {
+      projects: [],
+    };
+  }
+
+  componentDidMount() {
+    fetch(this.props.url + '/projects')
```
Comment (crisanlucid) [inline src/pages/Projects.js:20]: can be use 
`${this.props.url}/projects`

File: src/components/global/AddButton.js (hunk 3 above)
Comment (aliashkevich) [reply src/components/global/AddButton.js]: fixed

File: src/pages/Projects.js (hunk 9 above)
Comment (aliashkevich) [reply src/pages/Projects.js:20]: fixed

File: src/components/login/LoginForm.js (hunk 4 above)
Comment (jenna1k) [reply src/components/login/LoginForm.js:52]: I kept this comment to use it in the future but I can delete it if we are not gonna use it

**Pull request closed without merging**